docs/commands/commands.md

```diff
@@ -0,0 +1,67 @@
+# Command Cheat Sheet
+In this section, we'll describe `kubectl` commands to interact with Claudie.
+
+
+## Monitoring the cluster state
+Watch the cluster state in the `InputManifest` that is provisioned.
+```json
+watch -n 2 'kubectl get inputmanifests.claudie.io manifest-name -ojsonpath='{.status}' | jq .'
+{
+  "clusters": {
+    "my-super-cluster": {
+      "phase": "NONE",
+      "state": "DONE"
+    }
+  },
+  "state": "DONE"
+}   
+```
+
+## Viewing the cluster metadata
+  Each secret created by Claudie has following labels:
+
+  | Key                     | Value                                           |
+  | ----------------------- | ----------------------------------------------- |
+  | `claudie.io/project`    | Name of the project.                            |
+  | `claudie.io/cluster`    | Name of the cluster.                            |
+  | `claudie.io/cluster-id` | ID of the cluster.                              |
+  | `claudie.io/output`     | Output type, either `kubeconfig` or `metadata`. |
+
+Claudie creates kubeconfig secret in claudie namespace:
+
+  ```bash
+  kubectl get secrets -n claudie -l claudie.io/output=kubeconfig
+  ```
+  ```
+  NAME                                  TYPE     DATA   AGE
+  my-super-cluster-6ktx6rb-kubeconfig   Opaque   1      134m
+  ```
+
+  You can **recover kubeconfig** for your cluster with the following command:
+
+  ```bash
+  kubectl get secrets -n claudie -l claudie.io/output=kubeconfig -o jsonpath='{.items[0].data.kubeconfig}' | base64 -d > my-super-cluster-kubeconfig.yaml
```

Comment:
Hi @cloudziu what do you think about adding `claudie.io/cluster=<YOUR-CLUSTER-NAME>` to the label selection of this cmd? Because when you have multiple InputManifests in one namespace, a user might not get the wanted kubeconfig.

Reply:
If you agree with me, I think it will be reasonable to do that in all of the cmds.

Reply:
Will do